docs/development.md

```diff
@@ -0,0 +1,57 @@
+# Development
+
+This integration uses the
+[Heroku Platform API](https://devcenter.heroku.com/articles/platform-api-reference).
+The root of its entity and relationship ingestion begins with a
+[Heroku Enterprise Account](https://devcenter.heroku.com/categories/enterprise-accounts) -
+therefore, this integration currently offers no functionality for personal or
+professional accounts.
```

Comment:
I'm a bit confused... you say here that "this integration currently offers no functionality for personal or professional accounts" but the instructions for development include setting up a personal account. What am I missing here?

Reply:
That may be a _bit_ misleading. Technically you can run/test individual steps (like `fetch-team-members` and `fetch-team-apps`) using a normal user api key.

However, the `collect` step won't return any data with a normal user api key, since the root step is `fetch-enterprise-accounts`.